src/test/java/training/AppTest.java

```diff
@@ -1,14 +1,66 @@
-/*
- * This Java source file was generated by the Gradle 'init' task.
- */
-package training;
-
-import org.junit.Test;
-import static org.junit.Assert.*;
-
-public class AppTest {
-    @Test public void testAppHasAGreeting() {
-        App classUnderTest = new App();
-        assertNotNull("app should have a greeting", classUnderTest.getGreeting());
-    }
+  
+  package playground;
+
+class Shapes
+{
+  public static void main(String arg[])
+  {
+    Rectangle r = new Rectangle(10, 4);
+    Square s = new Square(7);
+    Circle c = new Circle(3.5);
+
+    System.out.println("Rectangle Area : " + r.getArea());
+    System.out.println("Square Area : " + s.getArea());
+    System.out.println("Circle Area : " + c.getArea());
+    System.out.println();
+
+
+  }
+}
+
+class Rectangle
```

Comment:
As I mentioned earlier please use inheritance and polymorphism to simplify the code.